source/views/help/wifi.js

```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
 
   render() {
     return (
-      <ScrollView>
-        <Card footer={this.state.status} header="Report a WiFi Problem!">
-          <Button
-            disabled={this.state.status !== ''}
-            onPress={this.start}
-            title="Report"
-          />
+      <View>
+        <Card footer={this.state.status}>
+          <Container>
+            <Title selectable={true}>Report a Wi-Fi Problem</Title>
+            <Description selectable={true}>
+              If you are having an issue connecting to any of the St. Olaf
+              College Wi-Fi networks, please tap the button below.
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
+              using, then send it to a server that IT maintains. They are
```

Comment:
>They are planning to look closely at where people are having issues!

We don't really speak on their behalf; I suppose I would be more neutral?  As in,

>The networking team can then use this information to identify where people are having issues!

Reply:
Also, we should be clear that this information is anonymous, and we do not collect usernames.

Reply:
I think it's okay to be slightly more wordy here just to be clear about what we're suggesting.  Alternatively, we could add some of this to the legal section and point people that way.

Reply:
Again, if we change this, let’s be sure the release notes read the same.